src/views/components/input/index.njk

```diff
@@ -0,0 +1,55 @@
+---
+title: Input
+description: The input component lets users enter alphanumeric characters for answers that are no longer than one line, using the appropriate keyboard on touch devices.
+layout: default
+eleventyNavigation:
+  key: Input
+  section: components
+---
+{% from "example.njk" import example %}
+
+<section>
+    <p>The input component lets users enter alphanumeric characters for answers that are no longer than one line, using the appropriate keyboard on touch devices.</p>
+    {{ example({group: "components", item: "input", example: "default", html: true, open: false }) }}
+</section>
+
+<section class="my-8">
+    <h2 class="mb-4">When to use this component</h2>
+    <p>Use the input component when you need to let users enter alphanumeric information that is no longer than one line, such as their telephone number or email address.</p>
```

Comment:
```suggestion
    <p>Use the input component when you need to let users enter alphanumeric information that is no longer than one line, such as their name, telephone number or email address.</p>
```

Reply:
Cheers, will update

Reply:
Have now updated the copy as noted in the suggestion